generator-mediator-js/app/templates/mediatorTemplate/lib/_index.js

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env node
+'use strict'
+
+const express = require('express')
+const medUtils = require('openhim-mediator-utils')
+
+const utils = require('./utils')
+const OpenHIM = require('./openhim')
+
+// Config
+var config = {} // this will vary depending on whats set in openhim-core
+const apiConf = require('../config/config')
+const mediatorConfig = require('../config/mediator')
+
+// socket config - large documents can cause machine to max files open
+const https = require('https')
+const http = require('http')
+
+https.globalAgent.maxSockets = 5
+http.globalAgent.maxSockets = 5
+
+var port = mediatorConfig.endpoints[0].port
+
+/**
+ * setupApp - configures the http server for this mediator
+ *
+ * @return {express.App}  the configured http server
+ */
+function setupApp () {
+  const app = express()
+
+  app.get(mediatorConfig.endpoints[0].path, (req, res) => {
+    console.log('Processing GET request on %s', mediatorConfig.endpoints[0].path)
+    var responseBody = 'Primary Route Reached'
+    var headers = { 'content-type': 'application/json' }
+
```

Comment:
I think have a comment here explaining that this is where the developer can add logic to alter the request.